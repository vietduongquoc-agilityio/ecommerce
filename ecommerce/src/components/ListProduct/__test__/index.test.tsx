import React from "react";
import { render } from "@testing-library/react";
import ProductList from "../index";

describe("ProductList Snapshot Test", () => {
  it("renders correctly and matches snapshot", () => {
    const { asFragment } = render(<ProductList />);
    expect(asFragment()).toMatchSnapshot();
  });
});
