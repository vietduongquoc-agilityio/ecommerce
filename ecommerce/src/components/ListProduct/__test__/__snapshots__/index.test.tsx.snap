// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductList Snapshot Test renders correctly and matches snapshot 1`] = `
<DocumentFragment>
  <p>
    Loading products...
  </p>
</DocumentFragment>
`;
